src/Tools/Vsdconfig/Vsdconfig.targets

```diff
@@ -5,7 +5,7 @@
           Inputs="@(VsdConfigXml);$(IntermediateOutputPath)\$(AssemblyName).dll"
           Outputs="$(OutDir)\$(AssemblyName).vsdconfig"
           Condition="'$(SkipGenerateVsdconfig)' != 'true'">
-    <Exec Command="&quot;$(VSSDK140Install)VisualStudioIntegration\Tools\Bin\vsdconfigtool.exe&quot; @(VsdConfigXml) $(IntermediateOutputPath)\$(AssemblyName).dll &quot;$(OutDir)\$(AssemblyName).vsdconfig&quot;" />
+    <Exec Command="&quot;$(VS140COMNTOOLS)..\..\VSSDK\VisualStudioIntegration\Tools\Bin\vsdconfigtool.exe&quot; @(VsdConfigXml) $(IntermediateOutputPath)\$(AssemblyName).dll &quot;$(OutDir)\$(AssemblyName).vsdconfig&quot;" />
```

Comment:
I had VSSDK140Install not show up on one machine for a day, and then it was back after a reboot.  Not sure why.